client/components/OperationList.js

```diff
@@ -71,8 +71,12 @@ class CategorySelectComponent extends React.Component {
     }
 }
 
-function ComputeAttachmentLink(opId) {
-    return `operations/${opId}/file`;
+function ComputeAttachmentLink(op) {
+    let file = 'file';
+    if (op.binary.fileName !== null) {
```

Comment:
Oops, I was confused with `undefined`, which should also probably be tested here. A quicker test (but error-prone, if the fileName variable is falsy) would be `let file = op.binary.fileName || 'file';`, which is fine to me too.